mivs/templates/mivs_judging/index.html

```diff
@@ -16,7 +16,7 @@
 <tbody>
 {% for game in games %}
     <tr>
```

Comment:
made a bug to fix this later because it's not that critical for right now https://github.com/magfest/mivs/issues/4